src/sentry/conf/types/uptime.py

```diff
@@ -0,0 +1,11 @@
+import dataclasses
+
+from sentry.conf.types.kafka_definition import Topic
+
+
+@dataclasses.dataclass
+class UptimeRegionConfig:
+    slug: str
+    name: str
+    config_topic: Topic
+    enabled: bool
```

Comment:
```suggestion
class UptimeRegionConfig:
    """Defines an available region which an uptime-checker is run in."""
    
    slug: str
    name: str
    config_topic: Topic
    enabled: bool
```

Wouldn't mind a comment here for other people reading these